### PR TITLE
#1565

### PR DESCRIPTION
**Pull request by michaelDpierce**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,18 +1,4 @@
 GIT
-  remote: git://github.com/capistrano/rvm.git
-  revision: dc35ec5e7a74a422250bd6ff5e8885c8b742d0e5
-  specs:
-    capistrano-rvm (0.1.1)
-      capistrano (~> 3.0)
-      sshkit (~> 1.2)
-
-GIT
-  remote: git://github.com/fnando/browser.git
```
Comment (duffcodester) [inline Gemfile.lock]: what was this used for?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -33,7 +31,6 @@
     </div>
     <%= render 'layouts/header' %>
     <div class='container'>
-      <% browser_check %>
```
Comment (duffcodester) [inline app/views/layouts/application.html.erb]: why aren't we using the browser check anymore?

File: Gemfile.lock (hunk 1 above)
Comment (michaelDpierce) [inline Gemfile.lock]: It was linked and compiled automatically by the capistrano gem.

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (michaelDpierce) [inline app/views/layouts/application.html.erb]: Seems irrelevant because the only one or two times this has been an issue with clients they intercom or email us directly. Thought it would be nice to remove the bloat.
